api/api_test.go

```diff
@@ -252,6 +252,22 @@ func TestCoinInfo(t *testing.T) {
 			expectedStatus: 200,
 			expectedBody:   "{\"circulating_supply\": 0, \"info\": {}, \"market_cap\": 0, \"total_supply\": 0, \"volume_24\": 0}",
 		},
+		{
+			name:      	    "test coin info not found",
+			requestUrl:     fmt.Sprintf("%s/v1/market/info?coin=500&token=ETHToken", server.URL),
+			requestMethod:  "GET",
+			requestBody:    "",
+			expectedStatus: 404,
+			expectedBody:   "{\"code\":404,\"error\":\"Coin info for coin id 500 (token: ETHToken, currency: USD) not found\"}",
+		},
+		{
+			name:           "test coin assets not found",
+			requestUrl:     fmt.Sprintf("%s/v1/market/info?coin=1000&token=ETHToken", server.URL),
+			requestMethod:  "GET",
+			requestBody:    "",
+			expectedStatus: 200,
+			expectedBody:   `{"circulating_supply": 0, "market_cap": 0, "total_supply": 0, "volume_24": 0}`,
+		},
```

Comment:
this shouldn't be part of this review. [This is already on master](https://github.com/trustwallet/watchmarket/blob/master/api/api_test.go#L255-L270). Can you please rebase before pushing?